src/Docker/Runner/StateFile.php

```diff
@@ -110,6 +110,11 @@ public function persistState(
         InputTableStateList $inputTableStateList,
         InputFileStateList $inputFileStateList,
     ): void {
+        $configurationId = $this->configurationId;
+        if (!$configurationId) {
+            return;
+        }
```

Comment:
Kdyz neni `configId` nastaveny, tak se ve skutecnosti vubec nevola `persistState`. Ale to PHPStan nevi a nelibilo se mu to.